contents/bogo_sort/code/julia/bogo.jl

```diff
@@ -9,15 +9,15 @@ function is_sorted(a::Vector{Float64})
     return true
 end
 
-function bogo_sort(a::Vector{Float64})
+function bogo_sort!(a::Vector{Float64})
     while(!is_sorted(a))
         shuffle!(a)
     end
 end
 
 function main()
     a = [1, 3, 2, 4]
```

Comment:
```suggestion
    a = [1.0, 3, 2, 4]
```

This is my fault, but there is a type error here where the bogo_sort function wants floats and we are reading in ints. Could we fix this?

Reply:
Done.